src/EditorFeatures/CSharpTest/Wrapping/InitializerExpressionWrappingTests.cs

```diff
@@ -0,0 +1,201 @@
+            ""the"",
+            ""lazy"",
+            ""dog""
+        };
+     }
+}",
+@"class C {
+    void Bar() {
+        var test = new[]
+        {
+            ""the"", ""quick"", ""brown"", ""fox"", ""jumps"", ""over"", ""the"", ""lazy"", ""dog""
+        };
+     }
+}"
+);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsWrapping)]
+        public async Task TestWrappingMultiLineLongIntializerExpression()
+        {
+            await TestAllWrappingCasesAsync(
+@"class C {
+    void Bar() {
+        var test = new[] [||]{ ""the"", ""quick"", ""brown"", ""fox"", ""jumps"", ""over"", ""the"", ""lazy"", ""dog"", ""the"", ""quick"", ""brown"", ""fox"", ""jumps"", ""over"", ""the"", ""lazy"", ""dog"" };
+     }
+}",
+@"class C {
+    void Bar() {
+        var test = new[]
+        {
+            ""the"",
+            ""quick"",
+            ""brown"",
+            ""fox"",
+            ""jumps"",
+            ""over"",
+            ""the"",
+            ""lazy"",
+            ""dog"",
+            ""the"",
+            ""quick"",
+            ""brown"",
+            ""fox"",
+            ""jumps"",
+            ""over"",
+            ""the"",
+            ""lazy"",
+            ""dog""
+        };
+     }
+}",
+@"class C {
+    void Bar() {
+        var test = new[]
+        {
+            ""the"", ""quick"", ""brown"", ""fox"", ""jumps"", ""over"", ""the"", ""lazy"", ""dog"", ""the"", ""quick"", ""brown"", ""fox"",
+            ""jumps"", ""over"", ""the"", ""lazy"", ""dog""
+        };
+     }
+}"
+);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsWrapping)]
+        public async Task TestShortInitializerExpressionRefactorings()
+        {
+            await TestAllWrappingCasesAsync(
+@"class C {
+    void Bar() {
+        var test = new[]
+        [||]{
+            1,
+            2
+        };
+    }
+}",
+@"class C {
+    void Bar() {
+        var test = new[] { 1, 2 };
+    }
+}",
+@"class C {
+    void Bar() {
+        var test = new[]
+        {
+            1, 2
+        };
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsWrapping)]
+        public async Task TestLongIntializerExpressionRefactorings()
+        {
+            await TestAllWrappingCasesAsync(
+@"class C {
+    void Bar() {
+        var test = new[]
+        [||]{
+            ""the"", ""quick"", ""brown"", ""fox"", ""jumps"", ""over"", ""the"", ""lazy"", ""dog""
+        };
+     }
+}",
+@"class C {
+    void Bar() {
+        var test = new[]
+        {
+            ""the"",
+            ""quick"",
+            ""brown"",
+            ""fox"",
+            ""jumps"",
+            ""over"",
+            ""the"",
+            ""lazy"",
+            ""dog""
+        };
+     }
+}",
+@"class C {
+    void Bar() {
+        var test = new[] { ""the"", ""quick"", ""brown"", ""fox"", ""jumps"", ""over"", ""the"", ""lazy"", ""dog"" };
+     }
+}"
+);
+        }
+    }
```

Comment:
would expect a test where an initializer is used in a few moew locations to demonstrate this is working.  Other examples:

1. fields,
2. arguments to method calls.
3. return statements.

Reply:
comment still applies

Reply:
Done, I added test for lists, properties, return, and arguments to method calls.

Reply:
awesome. t hanks!

Reply:
#resolved